src/Settings/settings.json

```diff
@@ -1182,5 +1182,25 @@
     "type": "bool",
     "default": false,
     "notr": true
+  },
+  {
+    "name": "WakaTime/Path",
+    "desc": "Path",
+    "type": "QString",
+    "default": "wakatime",
+    "ui": "PathItem",
+    "param": "PathItem::Executable",
+    "tip": "The path to the WakaTime executable file"
+  },
+  {
+    "name": "WakaTime/Api Key",
+    "desc": "Api Key",
```

Comment:
Could be elaborated, like where to find or URL with instructions to generate it.